### DISCUSSION TITLE
ACE NVG effects on AIs

### DISCUSSION THREAD

**Discussion by acknowledge14** [acemod/ACE3#9214] [Q&A] [answered]
ACE NVGs will heavily downgrade players' visibility and view distance, compared to vanilla ones. So do AI soliders wearing NVGs also be affectd by these heavily blured and noised green image and limited view distance?

ACE NVG also provide generation 1 to 4 presets, and several configurations(mono-, bi-, quadocular, wide view), which really has a effect on players. Do these differences also affect AIs?

https://ace3.acemod.org/wiki/feature/nightvision.html
https://ace3.acemod.org/wiki/framework/nightvision-framework.html

**Comment by BaerMitUmlaut** [comment, marked as answer]
No. We don't tweak AI and consider that the job of the mission maker, since any change would always be opinionated.

**Comment by RContreiras** [comment]
How does this work? Does the NVG need to stay on the AI's helmets or anywhere, like their backpacks?
Does it support all NVG mods or just vanilla ones?
I'm very intereted in this. Thanks in advance!

**Reply by RContreiras** [reply]
I did some testing
Test 1- AI starts wit NVG on his helmet in day time. As soon as i hit play, he hides it, but when i skip time to night, he does not put it back.

Test 2 - AI starts wit NVG on his helmet in night time. As soon as i hit play, he still has it, but when i skip time to day, he does not hide it.

What am i missing?

**Reply by BaerMitUmlaut** [reply]
The check runs every 5 minutes. This is also completely unrelated to the original topic.

**Reply by RContreiras** [reply]
Sorry, but i didnt want to start a new thread, because it was about the NVG with AI.
I read this in the updates note:
"- AI - Add automatic NVG equip/unequip (#9313)"

I want to know all about it
Does it work after the check?
Are modded NVGs supported?
I can make another thread if you want. Just let me know.

**Reply by jonpas** [reply]
All NVGs are supported, it does not modify inventory, only puts it on head and back in inventory based on time of day.

**Reply by RContreiras** [reply]
Nice! Thank you for the reply. Cheers!
